test_validation pytests not working as expected on macOS/arm64

Running `python -m pytest -q --tb=short --disable-warnings -k test_validation ` on the `main` branch on macOS/arm64 gives me the following failures:

Funnily enough, running the tests on Ubuntu/x86 gives me no failures.

```
=========================================================================================================== FAILURES ===========================================================================================================
___________________________________________________________________________________________________ test_missing_geopackage ____________________________________________________________________________________________________
tests/test_validation.py:34: in test_missing_geopackage
    assert False
E   assert False

During handling of the above exception, another exception occurred:
tests/test_validation.py:36: in test_missing_geopackage
    assert inst.__str__() == "Did not find geopackage file in ngen-run/config!!!"
E   AssertionError: assert 'assert False' == 'Did not find...run/config!!!'
E     
E     - Did not find geopackage file in ngen-run/config!!!
E     + assert False
---------------------------------------------------------------------------------------------------- Captured stderr setup -----------------------------------------------------------------------------------------------------
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 30.7M  100 30.7M    0     0  13.2M      0  0:00:02  0:00:02 --:--:-- 13.2M
----------------------------------------------------------------------------------------------------- Captured stdout call -----------------------------------------------------------------------------------------------------
Realization found! Retrieving catchment data...
Done
Validating /Users/qylee/Documents/NRDS/datastreamcli/tests/data/test_dir/ngen-run/config/realization.json
Done
Validating required individual catchment paths

NGen run folder is valid

__________________________________________________________________________________________________ test_duplicate_geopackage ___________________________________________________________________________________________________
tests/test_validation.py:44: in test_duplicate_geopackage
    assert False
E   assert False

During handling of the above exception, another exception occurred:
tests/test_validation.py:46: in test_duplicate_geopackage
    assert inst.__str__() == "This run directory contains more than a single geopackage file, remove all but one."
E   AssertionError: assert 'assert False' == 'This run dir... all but one.'
E     
E     - This run directory contains more than a single geopackage file, remove all but one.
E     + assert False
---------------------------------------------------------------------------------------------------- Captured stderr setup -----------------------------------------------------------------------------------------------------
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 30.7M  100 30.7M    0     0  6492k      0  0:00:04  0:00:04 --:--:-- 6873k
----------------------------------------------------------------------------------------------------- Captured stdout call -----------------------------------------------------------------------------------------------------
Realization found! Retrieving catchment data...
Done
Validating /Users/qylee/Documents/NRDS/datastreamcli/tests/data/test_dir/ngen-run/config/realization.json
Done
Validating required individual catchment paths

NGen run folder is valid

___________________________________________________________________________________________________ test_missing_realization ___________________________________________________________________________________________________
tests/test_validation.py:53: in test_missing_realization
    assert False
E   assert False

During handling of the above exception, another exception occurred:
tests/test_validation.py:55: in test_missing_realization
    assert inst.__str__() == "Did not find realization file in ngen-run/config!!!"
E   AssertionError: assert 'assert False' == 'Did not find...run/config!!!'
E     
E     - Did not find realization file in ngen-run/config!!!
E     + assert False
---------------------------------------------------------------------------------------------------- Captured stderr setup -----------------------------------------------------------------------------------------------------
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 30.7M  100 30.7M    0     0  9889k      0  0:00:03  0:00:03 --:--:-- 9890k
----------------------------------------------------------------------------------------------------- Captured stdout call -----------------------------------------------------------------------------------------------------
Realization found! Retrieving catchment data...
Done
Validating /Users/qylee/Documents/NRDS/datastreamcli/tests/data/test_dir/ngen-run/config/realization.json
Done
Validating required individual catchment paths

NGen run folder is valid

____________________________________________________________________________________________________ test_missing_forcings _____________________________________________________________________________________________________
tests/test_validation.py:82: in test_missing_forcings
    assert False
E   assert False

During handling of the above exception, another exception occurred:
tests/test_validation.py:84: in test_missing_forcings
    assert inst.__str__() == f"Forcings file not found!"
E   AssertionError: assert 'assert False' == 'Forcings file not found!'
E     
E     - Forcings file not found!
E     + assert False
---------------------------------------------------------------------------------------------------- Captured stderr setup -----------------------------------------------------------------------------------------------------
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 30.7M  100 30.7M    0     0  2020k      0  0:00:15  0:00:15 --:--:-- 2644k
----------------------------------------------------------------------------------------------------- Captured stdout call -----------------------------------------------------------------------------------------------------
Realization found! Retrieving catchment data...
Done
Validating /Users/qylee/Documents/NRDS/datastreamcli/tests/data/test_dir/ngen-run/config/realization.json
Done
Validating required individual catchment paths

NGen run folder is valid
```

--- a/tests/test_validation.py
+++ b/tests/test_validation.py
@@ -15,17 +15,15 @@ TEST_DATA_DIR = TEST_DIR / "ngen-run"
 
 @pytest.fixture(autouse=True)
 def ready_test_folder():
-    # download tarred file once only
-    if not ORIGINAL_TAR_PATH.exists():
-        response = requests.get(DATA_PACKAGE, stream=True, timeout=10)
-        response.raise_for_status()
-        with open(ORIGINAL_TAR_PATH, 'wb') as f:
-            for chunk in response.iter_content():
-                f.write(chunk)
-
     if TEST_DIR.exists():
         shutil.rmtree(TEST_DIR)
     TEST_DIR.mkdir(parents=True, exist_ok=True)
+
+    response = requests.get(DATA_PACKAGE, stream=True, timeout=10)
+    response.raise_for_status()
+    with open(ORIGINAL_TAR_PATH, 'wb') as f:
+        for chunk in response.iter_content():
+            f.write(chunk)
 
     with tarfile.open(ORIGINAL_TAR_PATH, 'r:gz') as tar:
         tar.extractall(path=TEST_DIR)

